repositories.json

```diff
@@ -109,5 +109,17 @@
           "-C": "3_submit"
       },
       "base": "challenge-aido_LF-template-ros"
-  }  
+  }, 
+  {
+      "name": "gym-duckietown",
+      "origin": "duckietown/gym-duckietown",
+      "dts_args": {
+          "-C": "docker/AIDO1/server-python3"
+      }
+  }, 
+  {
+      "name": "challenge-aido_LF-simulator-gym",
+      "origin": "duckietown/challenge-aido_LF-simulator-gym",
+      "base": "duckietown/gym-duckietown-server-python3:daffy-aido4"
```

Comment:
@CourchesneA 
This is supposed to be the name of the job, not the artifact. Use `gym-duckietown` instead.